framework/src/executioners/Executioner.C

```diff
@@ -64,6 +64,8 @@ Executioner::Executioner(const InputParameters & parameters)
     _restart_file_base(getParam<FileNameNoExtension>("restart_file_base")),
     _verbose(getParam<bool>("verbose"))
 {
+  _fe_problem.getNonlinearSystemBase().setVerboseFlag(_verbose);
```

Comment:
Does this give different results than previously setting it from `FEProblemSolve`? I usually like to see less training of method calls

Reply:
No. I think the rule of thumb is to let the object having the valid parameters use the parameters. Here, `verbose` is a parameter of `Executioner`, so it is more appropriate to set this flag here.